bindings/rust/evmc-declare/src/lib.rs

```diff
@@ -345,7 +346,7 @@ fn build_execute_fn(names: &VMNameSet) -> proc_macro2::TokenStream {
             use evmc_vm::EvmcVm;
 
             // TODO: context is optional in case of the "precompiles" capability
-            if instance.is_null() || context.is_null() || msg.is_null() || (code.is_null() && code_size != 0) {
+            if instance.is_null() || host.is_null() || context.is_null() || msg.is_null() || (code.is_null() && code_size != 0) {
```

Comment:
Depending on https://github.com/ethereum/evmc/pull/427#discussion_r342330665 we may need to remove `context.is_null()` here.

Reply:
The `context` may always be null. VM never needs to dereference it.
Now the TODO comment should be applied to the `host` as in the case of "precompiles" it is not useful and may be null — this can be discussed further in https://github.com/ethereum/evmc/pull/350.

Reply:
Fixed the `context` part. Lets leave `host` for #350.